cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************

```

Comment:
Not sure this file needs to be included

Reply:
yes, this file is needed if you want to write any custom commands. There aren't any right now though